locales/ja_JP/LC_MESSAGES/duckduckgo.po

```diff
@@ -19,7 +19,7 @@ msgstr ""
 "Plural-Forms: nplurals=1; plural=0\n"
 
 msgid "!Bang Search Shortcuts"
-msgstr "検索ショートカット(!Bang)"
+msgstr "検索ショートカット"
```

Comment:
If possible, I think we should keep "bang" here, either in English or katakana. Perhaps we could make it clearer, for example like `(英語名：!Bang)` or `(通称：バング)` or something like that. What do you think?

Reply:
The expression Bang is not familiar in Japanese, and I think it is more natural not to.
Bangという表現は日本語ではなじみがなく、ない方が自然だと思います。

Reply:
Sorry, I think I wasn't clear. I agree "Bang" on its own is not familiar to Japanese people so I meant including it in addition to the Japanese translation, for example something like:

`検索ショートカット（英語名：!Bang）`
or
`検索ショートカット（通称：バング）`

This is because we don't have all our products 100% translated into other languages – some things are in English only – so Japanese users might see some other DuckDuckGo pages or marketing messages in English using the term "Bang". I thought if they know 検索ショートカット is also called !Bang/バング then that would help them understand any English-only pages in future.

Reply:
I see. If that's the case, I think `検索ショートカット（英語名：!Bang）` is easier to understand.

Reply:
Thank you – sounds good. If it becomes to long when it appears within pages, then we could go back to the Japanese only.